src/sdk/namespace_fs.js

```diff
@@ -1419,23 +1419,22 @@ class NamespaceFS {
                 const versioned_path = latest_ver_info && this._get_version_path(key, latest_ver_info.version_id_str);
                 const versioned_info = latest_ver_info && await this._get_version_info(fs_context, versioned_path);
 
-                gpfs_options = is_gpfs ?
```

Comment:
can you move it back here? I think we wanted to open it at the beginning of the function

Reply:
Hey can you move it back here?